instrumentation/servlet/servlet-5.0/testing/build.gradle.kts

```diff
@@ -1,10 +1,28 @@
 plugins {
-  id("otel.java-conventions")
+  id("otel.javaagent-testing")
 }
 
 dependencies {
   api(project(":testing-common"))
+  api(project(":instrumentation:servlet:servlet-common:javaagent"))
   api(project(":instrumentation:servlet:servlet-common:bootstrap"))
 
   compileOnly("jakarta.servlet:jakarta.servlet-api:5.0.0")
+
+  testInstrumentation(project(":instrumentation:jetty:jetty-11.0:javaagent"))
+
+  testImplementation(project(":instrumentation:servlet:servlet-5.0:testing"))
+
+  testLibrary("org.apache.tomcat.embed:tomcat-embed-core:10.0.0")
+  testLibrary("org.apache.tomcat.embed:tomcat-embed-jasper:10.0.0")
+
+  // Tomcat 10.1 requires Java 11
+  latestDepTestLibrary("org.apache.tomcat.embed:tomcat-embed-core:10.0.+")
+  latestDepTestLibrary("org.apache.tomcat.embed:tomcat-embed-jasper:10.0.+")
```

Comment:
might as well add one more module that would test tomcat 10.1 and later

Reply:
we could also split the tests by java version - would reduce the number of projects from 5 to 4?

Reply:
or maybe we can use this - and not need to copy all test classes?

```
val testLatestDeps = findProperty("testLatestDeps") as Boolean

if (testLatestDeps) {
  otelJava {
    minJavaVersionSupported.set(JavaVersion.VERSION_11)
  }
}
```